### PR TITLE
Create Rule_21-2.md

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section21/Rule_21-2.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** For purchased HW/steam in the proposed model, the baseline shall have the same number of pumps as proposed
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+**Schema Version:** 0.0.25
```
Comment (supriyagoel) [inline docs/section21/Rule_21-2.md:9]: Both sections  G3.1.1.1 & G3.1.1.3.1 do not have the requirement 'For purchased HW/steam in the proposed model, the baseline shall have the same number of pumps as proposed'
G3.1.1.3.4 On-Site Distribution Pumps- says 'All on-site distribution pumps shall be modeled in both the proposed design and base building design'

Do we need to update both the ruleset reference and the rule description?

File: docs/section21/Rule_21-2.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+**Data Lookup:** None  
+**Function Call:** 
+
+1. check_purchased_chw_hhw()
+
+
+**Applicability Checks:**
+
+- Check if P-RMR is modeled with purchased chilled water or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(P_RMR)`
+
```
Comment (supriyagoel) [inline docs/section21/Rule_21-2.md:27]: this rule is applicable is P RMDi is modeled with PCHW or PHW/Steam. linke 14 only mentions PHW/Steam and like 26 mentions all 3 but the check is being done for PHW and Steam only.
should consistently check for purchased heating and cooling.

File: docs/section21/Rule_21-2.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  - If P-RMR is not modeled with purchased hot water/steam, rule is not applicable to B-RMR: `if NOT purchased_chw_hhw_status_dict["PURCHASED_HEATING"]: RULE_NOT_APPLICABLE`
+
+  - Else, P-RMR is modeled with purchased hot water/steam, continue to rule logic: `else: CHECK_RULE_LOGIC`
+
+## Rule Logic:  
+
+
+**Rule Assertion:**
+
+- Case 1:  `UNDETERMINED and raise_message "P-RMR IS MODELED WITH PURCHASED HOT WATER OR STEAM. VERIFY B-RMR HAS THE SAME NUMBER OF PUMPS AS THE PROPOSED."`
```
Comment (supriyagoel) [inline docs/section21/Rule_21-2.md:37]: 'P RMDI is modeled with PHW, Steam or PCHW'

**Review by supriyagoel — CHANGES_REQUESTED** [review]
please see comments in line

File: docs/section21/Rule_21-2.md (hunk 1 above)
Comment (supriyagoel) [reply docs/section21/Rule_21-2.md:9]: SG to confirm that Section 22 has a corresponding rule.

Comment (KarenWGard) [reply docs/section21/Rule_21-2.md]: updated to G3.1.1.3.4

File: docs/section21/Rule_21-2.md (hunk 2 above)
Comment (KarenWGard) [reply docs/section21/Rule_21-2.md:27]: section 21 consistently deals only with heating systems.  I believe there should be an equivalent rule in section 21 that deals with purchased cooling

File: docs/section21/Rule_21-2.md (hunk 3 above)
Comment (KarenWGard) [reply docs/section21/Rule_21-2.md]: updated to read "P-RMI"

Is "RMDI" more correct?

File: docs/section21/Rule_21-2.md (hunk 1 above)
Comment (supriyagoel) [reply docs/section21/Rule_21-2.md]: SG confirmed that Section 22 has a rule which addresses this ffor PCHW. 
SG has also updates the rules tab for Section 21 rule test spreadsheet with correct ruleset reference.

File: docs/section21/Rule_21-2.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** For purchased HW/steam in the proposed model, the baseline shall have the same number of pumps as proposed
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+**Schema Version:** 0.0.25
+
+**Applicability:** All required data elements exist for B_RMR
```
Comment (supriyagoel) [inline docs/section21/Rule_21-2.md]: As discussed on the call, should use RMI instead of RMR

**Review by supriyagoel — APPROVED** [review]

**Review by weilixu — APPROVED** [review]

**Pull request merged**